budcamp/Homework3/Homework3/task2/TestRunner.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class TestRunner
+    {
+        private TestContainer testContainer;
+        private TestResults testResults;
+
+        int correctAnswers = 0;
+        string userAnswer = "";
+
+        public TestRunner(TestContainer testContainer)
+        {
+            this.testContainer = testContainer;
+            testResults = new TestResults();
+        }
+
+        public void RunTest(string topic)
+        {
+            List<Test> chosenTopic = testContainer.GetTestByTopic(topic);
+
+            foreach (Test test in chosenTopic)
+            {
+                test.PrintTest();
+
+                if (test.CheckAnswer(userAnswer))
```

Comment:
Where are you actually reading user answer??
`userAnswers` in your case is always empty string.

One way to implement this would be:

```
string userAnswer = Console.ReadLine();
if (test.CheckAnswer(userAnswer))
{
  correctAnswers++;
}
```